### PR TITLE
Add summary to msteams notification

### PR DESCRIPTION
**Pull request by romankubu**
Hi,
currently msteams notification does not support summary atribute from Message card. Result is that in Activity (inside MS Teams) you don't know what kind of alert you received and you need to klik on it. It is same for notification on phone etc.

Example (bottom one is without pull request, top one is with pull request):
image

### CODE REVIEW

**Comment by romankubu** [comment]
If i run `go run -tags=dev asset_generate.go` in `/assets` then those alerts are working well. How can i fix it?

File: docs/configuration.md [hunk 1]
```diff
@@ -736,6 +736,9 @@ Microsoft Teams notifications are sent via the [Incoming Webhooks](https://learn
 # Message title template.
 [ title: <tmpl_string> | default = '{{ template "msteams.default.title" . }}' ]
 
+# Message title template.
```
Comment (zhan9san) [inline docs/configuration.md:739]: I guess it's a typo

```suggestion
# Message summary template.
```

**Review by zhan9san — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (romankubu) [reply docs/configuration.md]: Thanks i missed that

**Review by gotjosh — APPROVED** [review]
LGTM

Thank you very much for your contribution.

**Comment by rgarcia89** [comment]
Someone knows if a new release will be created close by? This is a very unfavorable bug

**Pull request merged**